WootricSDK/WootricSDK/WTRApiClient.m

```diff
@@ -367,4 +412,27 @@ - (NSString *)percentEscapeString:(NSString *)string {
   return [result stringByReplacingOccurrencesOfString:@" " withString:@"+"];
 }
 
+-(NSString *)buildUniqueLinkAccountToken:(NSString *)accountToken endUserEmail:(NSString *)endUserEmail date:(NSTimeInterval)date randomString:(NSString *)randomString {
+  const char* str = [[NSString stringWithFormat:@"%@%@%.0f%@", accountToken, endUserEmail, date, randomString] UTF8String];
+  unsigned char result[CC_SHA256_DIGEST_LENGTH];
+  CC_SHA256(str, (int) strlen(str), result);
+  
+  NSMutableString *hash = [NSMutableString stringWithCapacity:CC_SHA256_DIGEST_LENGTH * 2];
```

Comment:
Are we certain it will never blow up giving the fact that it's allocated 2 times its size?